homeassistant/components/nuki/__init__.py

```diff
@@ -46,8 +63,112 @@ async def async_setup(hass, config):
 
 async def async_setup_entry(hass, entry):
     """Set up the Nuki entry."""
-    hass.async_create_task(
-        hass.config_entries.async_forward_entry_setup(entry, LOCK_DOMAIN)
+
+    hass.data.setdefault(DOMAIN, {})
+
+    try:
+        bridge = await hass.async_add_executor_job(
+            NukiBridge,
+            entry.data[CONF_HOST],
+            entry.data[CONF_TOKEN],
+            entry.data[CONF_PORT],
+            True,
+            DEFAULT_TIMEOUT,
+        )
+
+        locks, openers = await hass.async_add_executor_job(_get_bridge_devices, bridge)
+    except InvalidCredentialsException as err:
+        raise InvalidAuth from err
+    except RequestException as err:
+        raise CannotConnect from err
```

Comment:
That is only used inside the flow. Just doing
```suggestion
    except (InvalidCredentialsException, RequestException):
        return False
```